BraveShared/Analytics/DAU.swift

```diff
@@ -38,12 +39,17 @@ public class DAU {
 
   private static let apiKeyPlistKey = "API_KEY"
   private let apiKey: String?
+  private let braveCore: BraveCoreMain?
 
-  public init(date: Date = Date()) {
+  public init(
+    date: Date = Date(),
+    braveCore: BraveCoreMain?
```

Comment:
I wonder whether we need entire brave core passed here or only the `braveStats` part?

Reply:
Good call, we only need the stats part

Reply:
Updated to use stats object directly